modules/gke-hub/outputs.tf

```diff
@@ -0,0 +1,15 @@
+/**
```

Comment:
I would remove this if empty. But we probably want to have a cluster id output that depends on feature activation, WDYT?

Reply:
done